tests/vmi_lifecycle_test.go

```diff
@@ -34,6 +34,7 @@ import (
 	. "github.com/onsi/gomega"
 	. "github.com/onsi/gomega/gstruct"
 	k8sv1 "k8s.io/api/core/v1"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
```

Comment:
Since https://github.com/kubevirt/kubevirt/pull/6955 we updated the codebase to use `policy/v1`, please update this as well.

Reply:
Here it is correct to use `policy/v1beta1` since it is used to pass the correct type `policyv1beta1.Eviction{}` to the `EvictV1beta1()` method. 
`Eviction{}` type is present in both `v1` and `v1beta1` package; they are used respectively in `EvictV1()` and `EvictV1beta1()`. We cannot use `EvictV1()` since it was introduced in k8s version 1.22 and we have to be compatible also with version 1.21.